app.js

```diff
@@ -1,22 +1,30 @@
-const express = require('express')
-const bodyParser = require('body-parser')
+
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+const dotenv = require('dotenv');
 const logger = require('./logger');
-const mongoose = require('mongoose')
-const dotenv = require('dotenv')
+const app = express();
+const port = 3000;

```

Comment:
I know it is not part from your task, but it should be env variable.

Reply:
????

Reply:
שינתי את זה זה נמצא שורה אחת מיתחת